src/InvertedIndices.jl

```diff
@@ -9,7 +9,9 @@ struct InvertedIndex{S}
 end
 const Not = InvertedIndex
 # Support easily inverting multiple indices without a temporary array in Not([...])
-InvertedIndex(i₁::Integer, i₂::Integer, iₓ::Integer...) = InvertedIndex(TupleVector((i₁, i₂, iₓ...)))
+function InvertedIndex(i₁::T, i₂::T, iₓ::T...)  where T
```

Comment:
The right syntax is `InvertedIndex(idxs::T...) where {T}`.

@pdeffebach - do you feel like finishing this PR (adding tests, docs, etc. 😄)